Warsaw_Tram/AlertHelper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AlertHelper.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 25/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+
+protocol AlertHelperProtocol {
+    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction, presentingViewController: UIViewController)
+    func showAlert(title: String, message: String, okAction: UIAlertAction, presentingViewController: UIViewController)
+}
+
+class AlertHelper: NSObject, AlertHelperProtocol {
```

Comment:
I’ve done that because that’s how it was done in my previous project... but now when I think about it, it doesn’t have to inherit after anything. For the second comment: Do you mean that I should do it as a UIViewController extension?